<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveSnapShotDialog</class>
 <widget class="QDialog" name="SaveSnapShotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Snapshot</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout" name="hlayout1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output folder  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outDirLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseDir">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Base name  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="baseNameLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Counter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="counterSpinBox">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="alllayersCheckBox">
       <property name="text">
        <string>Snap All Layers</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="tiledSaveCheckBox">
       <property name="toolTip">
        <string>If checked, save each image independently, allowing to later combine the saved images into a very very large image</string>
       </property>
       <property name="text">
        <string>Tiled Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout3">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="backgroundComboBox">
       <item>
        <property name="text">
         <string>MeshLab Gradient</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Alpha Transparent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solid White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Solid Black</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>The resolution of the screenshot is the resolution of the current window multiplied by this number</string>
       </property>
       <property name="text">
        <string>Screen Multiplier  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="resolutionSpinBox">
       <property name="toolTip">
        <string>The resolution of the screenshot is the resolution of the current window multiplied by this number</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="addToRastersCheckBox">
       <property name="text">
        <string>Add Snapshot as new Raster Layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveSnapShotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveSnapShotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
